### PR TITLE
close #115: avoid the term instance

### LINKED ISSUE #115: (editorial): Be less lax with "instance" term (CH-1)

### PR DESCRIPTION
**Pull request by cabo**
...except in "for instance", and in the discussion of default values.

### CODE REVIEW

File: sdf.md [hunk 1]
```diff
@@ -807,7 +813,15 @@ This example defines an Object "foo" that is defined in the default namespace (f
 `#/sdfObject/foo/sdfProperty/bar`, with data of type boolean.
 <!-- we could define a URN-style namespace that looks exactly that way -->
 
-Some of the definitions are also declarations: the definition of the entry "bar" in the property "foo" means that each instance of a "foo" can have zero or one instance of a "bar".  Entries within `sdfProperty`, `sdfAction`, and `sdfEvent`, within `sdfObject` entries, are declarations.  Similarly, entries within an `sdfThing` describe instances of `sdfObject` (or nested `sdfThing`) that form part of instances of the Thing.
+Some of the definitions are also declarations: the definition of the
```
Comment (akeranen) [inline sdf.md:816]: Given declarations are the norm, would it make sense to say “Often the definitions are…”.  and note the exception here explicitly?

**Review by akeranen — APPROVED** [review]
I think these are very good improvements. A few comments on potential terminology tweaking but good to go as such too.

File: sdf.md (hunk 1 above)
Comment (cabo) [reply sdf.md]: Great point.  See commit below.

File: sdf.md [hunk 2]
```diff
@@ -1348,8 +1362,8 @@ quality is absent.
 | sdfAction   | action    | zero or more named action definitions for this Object                    |
 | sdfEvent    | event     | zero or more named event definitions for this Object                     |
 | sdfData     | named-sdq | zero or more named data type definitions that might be used in the above |
-| minItems    | number    | (array) Minimum number of sdfObject instances in array                   |
-| maxItems    | number    | (array) Maximum number of sdfObject instances in array                   |
+| minItems    | number    | (array) Minimum number of multiplied affordances in array |
```
[the remark this answers is not in the document]
Comment (cabo) [reply sdf.md:1365]: Indeed.  I also didn't manage to find a better term.

[the remark this answers is not in the document]
Comment (JKRhb) [reply sdf.md:1365]: Hmm, how about "repeated" or "duplicated"?

File: sdf.md (hunk 1 above)
Comment (akeranen) [reply sdf.md]: Great, thanks!

**Comment by WAvdBeek** [comment]
i think we should fight back on not using the term instance.

[An instance is one occurrence of a class or object in programming](https://www.bing.com/ck/a?!&&p=d057176ef1b27f5cJmltdHM9MTY5NTk0NTYwMCZpZ3VpZD0zYzZkZGY1ZS0xYjliLTZlOGMtMzFjOS1jZDM4MWFiYzZmNjQmaW5zaWQ9NTc5NA&ptn=3&hsh=3&fclid=3c6ddf5e-1b9b-6e8c-31c9-cd381abc6f64&psq=what+is+an+instance+in+computer+language&u=a1aHR0cHM6Ly93d3cuY29tcHV0ZXJob3BlLmNvbS9qYXJnb24vaS9pbnN0YW5jZS5odG0&ntb=1)

File: sdf.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (JKRhb) [reply sdf.md:1365]: Hmm, simply speaking of "sdfObjects" (or maybe even "identical sdfObjects"?) wouldn't work here either, I assume?

```suggestion
| minItems    | number    | (array) Minimum number of sdfObjects in array |
```

If so, then using "multiplied" is more than good enough I'd say :)

[the remark this answers is not in the document]
Comment (cabo) [reply sdf.md:1365]: Well, it is the single sdfObject that has its guts multiplied, not additional sdfObjects added.
I'm not necessarily a great fan of not being able to call the single items something, but that is what the current structure does.

File: sdf.md [hunk 3]
```diff
@@ -807,7 +813,15 @@ This example defines an Object "foo" that is defined in the default namespace (f
 `#/sdfObject/foo/sdfProperty/bar`, with data of type boolean.
 <!-- we could define a URN-style namespace that looks exactly that way -->
 
-Some of the definitions are also declarations: the definition of the entry "bar" in the property "foo" means that each instance of a "foo" can have zero or one instance of a "bar".  Entries within `sdfProperty`, `sdfAction`, and `sdfEvent`, within `sdfObject` entries, are declarations.  Similarly, entries within an `sdfThing` describe instances of `sdfObject` (or nested `sdfThing`) that form part of instances of the Thing.
+Often, definitions are also declarations: the definition of the
+entry "bar" in the property "foo" means that data corresponding to the
+"foo" property in a property interaction offered by Thing can have zero or
+one components modeled by "bar".  Entries within `sdfProperty`,
+`sdfAction`, and `sdfEvent`, within `sdfObject` entries, are
```
[the remark this answers is not in the document]
Comment (cabo) [reply sdf.md:820]: Good point!

[the remark this answers is not in the document]
Comment (cabo) [reply sdf.md]: Text added to [c3d5759](https://github.com/ietf-wg-asdf/SDF/pull/132/commits/c3d5759547db7b25be4cf26e511d0a22a4b02753)

**Review by JKRhb — APPROVED** [review]

**Comment by cabo** [comment]
> i think we should fight back on not using the term instance.
> 
> [An instance is one occurrence of a class or object in programming](https://www.bing.com/ck/a?!&&p=d057176ef1b27f5cJmltdHM9MTY5NTk0NTYwMCZpZ3VpZD0zYzZkZGY1ZS0xYjliLTZlOGMtMzFjOS1jZDM4MWFiYzZmNjQmaW5zaWQ9NTc5NA&ptn=3&hsh=3&fclid=3c6ddf5e-1b9b-6e8c-31c9-cd381abc6f64&psq=what+is+an+instance+in+computer+language&u=a1aHR0cHM6Ly93d3cuY29tcHV0ZXJob3BlLmNvbS9qYXJnb24vaS9pbnN0YW5jZS5odG0&ntb=1)

Indeed, instance is a perfectly cromulent term.  The problem is that we were using it for different things, and entirely replacing it by different language reduces confusion (and we have the term available for instances of the model!).

**Pull request merged**